make-ltdb.bash

```diff
@@ -161,11 +161,12 @@ then
     unset DISPLAY;
     unset LUI;
 
-LISP="
-  ${extralisp}
-  (format t \"~%LTDB Read Grammar~%\")
+LISP="""
```

Comment:
I don't think bash has triple-quoted (""") strings. This might work anyway if it's evaluated as an empty string "" followed by a multiline string.